### DISCUSSION TITLE
Scythe Anticheat v2.14.0

### DISCUSSION THREAD

**Discussion by MrDiamond64** [Scythe-Anticheat/Scythe-Anticheat#99] [Announcements]
Scythe Anticheat is now at version v2.14.0!

Supported Minecraft Versions: 1.20.0

# Changelog
- Add support for minecraft version1.20.0
- Freeze uses /inputpermission
- Bug fixes
- Performance optimizations

<hr /><em>This discussion was created from the release <a href='https://github.com/Scythe-Anticheat/Scythe-AntiCheat/releases/tag/'>Scythe Anticheat v2.14.0</a>.</em>

**Comment by fahribeyyss1123445** [comment]
Hi I'm Fahri Can you give me permission to Edit Anti Cheatin Please my friend you very much. I love you but can you please give me license permission and don't insult me ​​:/ I'm only 13 please wait

**Reply by MrDiamond64** [reply]
You don't need permission to edit the addon. Anyone can edit it.

If you want to know what you can or can not do, I would advise you to read the license.
